CHANGELOG.md

```diff
@@ -26,7 +26,7 @@
 
 - Removed: support for loading config path using the `--config` flag. cosmiconfig will not parse command line arguments. Your application can parse command line arguments and pass them to cosmiconfig.
 - Removed: `argv` config option.
-- Removed: support for Node versions < 4.
+- Removed: support for Node versions &lt; 4.
```

Comment:
Is it possible to use the `<` symbol instead of `&lt;`? Seems unnecessary..

Reply:
This is automatically changed by remark-cli, so I'm not going to worry about it.